drivers/clock_control/clock_control_nrfxlib.c

```diff
@@ -9,8 +9,13 @@
 #include <device.h>
 #include <kernel_includes.h>
 #include <clock_control.h>
+#ifdef CONFIG_BT_LL_NRFXLIB
 #include <ble_controller.h>
 #include <ble_controller_soc.h>
+#else /* CONFIG_MPSL_LIBRARY */
+#include <mpsl.h>
+#include <mpsl_clock.h>
+#endif
```

Comment:
Eventually, clock_control_nrfxlib will integrate only with MPSL, and not both MPSL and ble_controller, right?

Reply:
Yes, that's the plan.